Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -418,10 +418,9 @@ final class ChatViewController: SLKTextViewController {
                     var insert = true
 
                     // swiftlint:disable for_where
```

Comment:
Can you remove this `swiftlint` comment?

Reply:
Done. 👍